compliance/google/long_stopped_instances/README.md

```diff
@@ -8,27 +8,37 @@ This policy checks all Google instances that are stopped and reports on any that
 
 The policy leverages the Google API to check all instances that have been stopped for longer than the specified period. If the action is approved, the instance is terminated.
 
-### Input Parameters
+## Input Parameters
 
 - *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
 - *Stopped days* - Number of days an instance is stopped before it is added to the report
-- *Google Cloud Project* -Google cloud project name
 
-### Required RightScale Roles
-
-- credential_viewer
-
-### Actions
+## Actions
 
 The following policy actions are taken on any resources found to be out of compliance.
 
 - Send an email report
 - Delete all instances after approval
 
-### Supported Clouds
+## Prerequisites
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+
+### Credential configuration
+
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
+
+Provider tag value to match this policy: `gce`
+
+Required permissions in the provider:
+
+- The `Monitoring Viewer` Role, the `compute.instances.delete`, `compute.instances.list` Permissions
```

Comment:
add a bullet for each permission/role

Reply:
I have addressed this comment please review the changes.